Clients/Xamarin.Interactive.Client.Mac/AppDelegate.cs

```diff
@@ -96,21 +97,33 @@ public override void WillFinishLaunching (NSNotification notification)
         {
             MenuManager = new MenuManager (NSApplication.SharedApplication.MainMenu);
 
-            ClientApp.SharedInstance.Updater.CheckForUpdatesPeriodicallyInBackground (
-                update => UpdateHandler (null, update));
+            if (ClientInfo.Flavor != ClientFlavor.Inspector)
```

Comment:
I'm not necessarily against this (and related subsequent checks), but it makes it impossible to update the non-IDE-bundled workbooks/inspector installation.

Reply:
Yeah, I thought about it and decided that was fine. More consistent, too.